src/renderer/store/modules/settings.js

```diff
@@ -167,7 +167,7 @@ const state = {
   barColor: false,
   checkForBlogPosts: true,
   checkForUpdates: true,
-  baseTheme: 'dark',
+  baseTheme: 'system',
```

Comment:
I have doubt of whether we should update the default to `system`

Reply:
Initially this was set to dark on the whim that we don't want to blind dark theme users, I think setting it to system is the reasonable default as it simply takes the nativeTheme dark theme preference (most operating systems now have some sort of switch, with people using whatever they prefer).

The alternatives here are:

* Keep it set at "dark", which then means light theme users have to switch away
* Set it to "light" and feel the wrath of all the dark theme users coming into github issues complaining
* Set it to "system" and let the NativeTheme handle what the theme should start as

I think system is the more seamless option as it respects the users system from the first launch, not married to it at all so if after this you still have doubts let's keep it at dark :)

Reply:
System works perfect!